dev-util/ostree/ostree-2020.6.ebuild

```diff
@@ -69,7 +69,6 @@ src_configure() {
 		$(use_with curl)
 		$(use_with dracut)
 		$(use_enable doc gtk-doc)
-		$(use_enable introspection)
```

Comment:
The gentoo dev manual states that "autodetection must be overridden":

https://devmanual.gentoo.org/general-concepts/use-flags/index.html

Reply:
Updated in 563bd09fe946701aba0b43a11bdfad5059561d67.